atlp-rwanda/eagles-ec-fe
# #187419123-Feature Users should be able to Update their Password

## what Does this PR do?
- Logged user are able to update their password
## How to test the functionality ?
- Checkout to the preview deployment of this PR
- checkout to the ||   URL/login (login is prerequisite to be able to update the PW)
- checkout to the ||  URL/update-password
- follow the procedure and update the password 
- verify if the password has been updated by perhaps trying to log in again
